ldk/javascript/packages/create-loop/index.js

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs/promises');
+const path = require('path');
+const { execSync } = require('child_process');
+const prompts = require('prompts');
+const Sqrl = require('squirrelly');
+
+const projectOptions = {
+  name: '',
+  language: '',
+  aptitudes: [],
+};
+
+const installNodeModules = async () => {
```

Comment:
i dont think this function needs to be async

Reply:
Good call!